cargo-apk/README.md

```diff
@@ -156,8 +156,13 @@ orientation = "landscape"
 
 # See https://developer.android.com/guide/topics/manifest/activity-element#exported
 #
-# Unset by default, or "true" when targeting Android >= 31 (S and up).
-exported = "true"
+# Unset by default, or true when targeting Android >= 31 (S and up).
+exported = true
+
+# See https://developer.android.com/guide/topics/manifest/activity-element#resizeableActivity
+#
+# Defaults to true on Android >= 24, no effect on earlier API levels
```

Comment:
There's another detail on Android >=31, but it seems a bit much to describe here.